src/Http/Client/Auth/Digest.php

```diff
@@ -83,7 +83,7 @@ protected function _getServerInfo(Request $request, $credentials)
         $response = $this->_client->get(
             $request->getUri(),
             [],
-            ['auth' => []]
+            ['auth' => ['type' => 'basic']]
```

Comment:
Why do you need to switch to basic auth in order to get the digest challenge information?

Reply:
Digest requires an initial hit to the server if the realm, nonce etc are not known.  This doesn't have to be a digest hit - in fact any unauthenticated hit will give this in formation in the response headers (as I undestand it at least).  If it is left as digest, then that initial server hit triggers another initial server hit... which keeps going forever.  It could be left as digest if we could somehow break this loop.  Given the fact that "blank" auth (as was used here) defaults to basic, this simply ensures the code works as originally intended.

Reply:
OK. Does setting `auth => false` disable the authentication headers entirely? My concern is trying to negotiate as basic auth and the server not responding with challenge headers.

Reply:
Might need to do `auth => null` instead. `false` looks like it will result in errors.

Reply:
ok let me check. sorry for the delay

Reply:
OK.  I can get it to work on auth => null.  Will update the pull request.